src/components/SingleReview.jsx

```diff
@@ -1,14 +1,27 @@
 import { useEffect } from "react"
 import { useState } from "react"
 import { useParams } from "react-router"
-import { getSingleReview } from "../api"
+import { getSingleReview, patchVotes } from "../api"
 import { Comments } from "./Comments"
 
 export const SingleReview = ()=> {
     const [isLoading, setIsloading] = useState(true)
     const [review, setReview] = useState([])
+    const [voteChange, setVoteChange] = useState(0)
+
     const {review_id} = useParams()
 
+    const voteChanger = (num) => {
+        setVoteChange((currVoteChange) => {
+            return (currVoteChange + num)})
+
+        patchVotes(review_id, num).catch(() =>{
+            setVoteChange((currVoteChange) => {
+                return (currVoteChange - num)})
+            console.error('Ooops something went wrong!');
```

Comment:
Console.error would work for a dev looking at this but for a user it would be useful to see an error message around the like button if the request fails